src/main/java/com/checkout/common/beta/CustomerRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.checkout.common.beta;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class CustomerRequest {
```

Comment:
That's not possible because `CreateCustomerInstrumentRequest` extends it. The idea is to be able to use this class for final or inherited purposes. There's a big number of `CustomerRequest` classes, and definitely this has to be fixed. This is kind of the first step to have a little bit more organization around `CustomerRequest` and `CustomerResponse`, for example. Wdyt?

Reply:
Ah I see, that's a reasonable use case, cool I'm okay with that.